src/app/stacktrace/stacktrace.module.ts

```diff
@@ -1,12 +1,18 @@
-import { NgModule } from '@angular/core';
-import { CommonModule } from '@angular/common';
+// import { NgModule } from '@angular/core';
+// import { CommonModule } from '@angular/common';
+// import { StacktraceHomeComponent } from './components/stacktrace-home/stacktrace-home.component';
+// import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+// import { StacktraceRoutingModule } from './stacktrace-routing.module';
 
-
-
-@NgModule({
-  declarations: [],
-  imports: [
-    CommonModule
-  ]
-})
-export class StacktraceModule { }
+// @NgModule({
+//   declarations: [StacktraceHomeComponent],
+//   imports: [
+//     CommonModule,
+//     NgbModule,
+//     StacktraceRoutingModule
+//   ],
+//   exports: [
+//     StacktraceHomeComponent
+//   ]
+// })
+// export class StacktraceModule { }
```

Comment:
Module changes might conflict with routing update, maybe we should wait for #48 first.